repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/servlet/http/HttpServletRequestWrapper.scala

```diff
@@ -103,29 +88,31 @@ class HttpServletRequestWrapper(originalRequest: HttpServletRequest, inputStream
     }
   }
 
+  override def addHeader(headerName: String, headerValue: String): Unit = {
+    val existingHeaders: List[String] = getHeadersScala(headerName) //this has to be done before we remove from the list,
+    // because getting this list is partially based on the contents of the removed list
+    removedHeaders = removedHeaders - headerName
+    headerMap = headerMap + (headerName -> (existingHeaders :+ headerValue))
+  }
+
```

Comment:
That's a really good question. I did not intend to, but IntelliJ's auto-formatting does not like making up its mind.

I suppose I can manually fix this up, but it gets incredibly annoying anytime I touch this file since I can't really use the auto-formatter.